src/ingest_validation_tools/table-schemas/sample.yaml

```
@@ -98,6 +98,7 @@ fields:
       - Freezer (-80 Celsius)
       - Freezer (-20 Celsius)
       - Room Temperature
+      - 4C
```

```suggestion
      -  Refrigerator (4 Celsius)
```

Is that correct? The options should follow the same pattern if possible.

---

Sounds good.